src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -808,78 +808,93 @@ g.test('depth_stencil_attachment,sample_counts_mismatch')
     }
   });
 
-g.test('depth_stencil_attachment')
+g.test('depth_stencil_attachment,loadOp_storeOp_match_depthReadOnly_stencilReadOnly')
   .desc(
     `
   Test GPURenderPassDepthStencilAttachment Usage:
     - depthReadOnly and stencilReadOnly must match if the format is a combined depth-stencil format.
-    - depthLoadOp and depthStoreOp must be provided iff the format has a depth aspect and
-      depthReadOnly is not true.
-    - stencilLoadOp and stencilStoreOp must be provided iff the format has a stencil aspect and
-      stencilReadOnly is not true.
+    - depthLoadOp and depthStoreOp must be provided if the format has a depth aspect and
```

Comment:
I agree with the iff change, but I think it might just be clearer to have break these points up a bit (even if it is a little more verbose) like the spec, (applies for stencil part also) i.e:

- if the format has a depth aspect:
    - if depthReadOnly is true:
        -  depthLoadOp and depthStoreOp must be provided
    - else:
        - depthLoadOp and depthStoreOp must not be provided

Reply:
done